pmix_governance.tex

```diff
@@ -234,6 +231,20 @@ \subsubsection{ASC Meetings}
 Standard up for consideration during that meeting in the published
 agenda.
 
+ASC quarterly meeting agenda items must be submitted to the Co-Chairs
+at least four weeks in advance of the meeting. Four weeks in advance of
+the meeting the Co-Chairs will announce the tentative agenda for review.

```

Comment:
Just a minor comment - this puts a lot of strain on the chairs: technically someone could submit an item in the last second before the deadline and then the chairs are still expected to immediately have this on the agenda. I would suggest to leave a small gap between the submission and the agenda deadline

Reply:
Informally we give ourselves about a day to post the revised agenda after the deadline. But we could formalize that.

Reply:
I'd advise keeping this flexible - the more rigid you try to make it, the more time you are going to waste trying to specify everything down to the tiniest detail (e.g., what time of day are you closing the agenda, what time zone, ...)

Reply:
Flexibility is fine and we are handling this in MPI the same way (we never specified timezones and used common sense), but everyone also respected and if it came a day after, then it was not on the agenda. However, the text above is exactly not flexible - it says that the agenda will be posted at the same time the last elements can be submitted - my suggestion would be at least 4 weeks for submission of items, at least 3 weeks for publishing agenda and at least 2 weeks for final agenda - if chairs are good and publish it at the 4 week mark, that's great then

Reply:
Per conversation during the 3Q 2020 ASC meeting:
 * The sense of the group was to keep this flexible.
 * If there is need to make it more specific then someone could do that in another PR.